packages/frontend/src/hooks/dashboard/useDashboard.tsx

```diff
@@ -83,6 +92,37 @@ export const useDashboardQuery = (
     });
 };
 
+export const useExportDashboard = () => {
+    const { showToastSuccess, showToastError, showToast } = useToaster();
+    return useMutation<string, ApiError, Dashboard>(
+        (data) => exportDashboard(data.uuid),
+        {
+            mutationKey: ['export_dashboard'],
+            onMutate: (data) => {
+                showToast({
+                    key: 'dashboard_export_toast',
+                    title: `${data.name} is being exported.`,
```

Comment:
Would be nice for this to be blue ( `intent: Intent.PRIMARY` ) and have a spinner or say `This might take a few seconds` 
![Captura de ecrã 2023-03-15, às 17 21 54](https://user-images.githubusercontent.com/9117144/225390979-50a386ac-9aee-4e7b-9133-ea19b374507a.png)